src/object/elo.c

```diff
@@ -350,6 +350,85 @@ elo_copy (DB_ELO * elo, DB_ELO * dest)
   return ret;
 }
 
+/*
+ * elo_copy_with_suffix () - Similar to the LOB_PERMANENT_CREATED case in elo_copy(),
+                             but called when adding a suffix to the destination path during copy.
+ * return: error code
+ * elo(in): DB_ELO structure that represents the original source file
+ * dest(out): DB_ELO structure that represents the copied file
+ * suffix(in): suffix that will be added to the destination path when copying
+ */
+int
+elo_copy_with_suffix (DB_ELO * elo, DB_ELO * dest, const char *suffix)
+{
+  int ret = NO_ERROR;
+  ES_URI out_uri, real_locator;
+  char *locator = NULL;
+  char *meta_data = NULL;
+
+  assert (elo != NULL);
+  assert (dest != NULL);
+  assert (elo->type == ELO_FBO);
+  assert (elo->locator != NULL);
+
+  /* create elo instance and copy file */
+  if (elo->meta_data != NULL)
+    {
+      meta_data = db_private_strdup (NULL, elo->meta_data);
+      if (meta_data == NULL)
+	{
+	  assert (er_errid () != NO_ERROR);
+	  return er_errid ();
+	}
+    }
+
+  /* if it uses external storage, do transaction work */
+  elo->es_type = es_get_type (elo->locator);
+  if (elo->es_type == ES_POSIX)
+    {
+      strcpy (real_locator, elo->locator);
+
+      ret = es_copy_file_with_suffix (real_locator, elo->meta_data, out_uri, suffix);
```

Comment:
strcpy (real_locator, elo->locator); 및 real_locator 제거 가능

```suggestion
      ret = es_copy_file_with_suffix (elo->locator, elo->meta_data, out_uri, suffix);
```

Reply:
그렇네요. 검증해보고 수정하겠습니다.

Reply:
수정하였습니다.